src/en/howto-juju-block.md

```diff
@@ -0,0 +1,105 @@
+# Restricting changes to the running Juju environment
```

Comment:
You should add a title. The metadata gets picked up by the build and is used as the actual page title in the browser. The first line just needs to be something like :

Title: Using Juju block to prevent accidental changes